mod_conf_sql.c

```diff
@@ -78,9 +75,9 @@ struct {
 #define SQLCONF_DEFAULT_CONF_ID_NAME    "conf_id"
 #define SQLCONF_DEFAULT_CTXT_ID_NAME    "ctxt_id"
 #define SQLCONF_DEFAULT_ID_NAME         "id"
-#define SQLCONF_DEFAULT_KEY_NAME       	"key"
+#define SQLCONF_DEFAULT_KEY_NAME        "type"
 #define SQLCONF_DEFAULT_PARENT_ID_NAME  "parent_id"
-#define SQLCONF_DEFAULT_VALUE_NAME      "value"
+#define SQLCONF_DEFAULT_VALUE_NAME      "info"
```

Comment:
Hmm.  These column name changes would break backward compatibility for existing installations; why change them?